docs/develop/Architecture.md

```diff
@@ -0,0 +1,77 @@
+# Architecture
+
+There are five repositories in the ElectionGuard SDK.
+
+- [ElectionGuard Specification][election-guard-specification]
+- [ElectionGuard Python][election-guard-python-source]
+- [ElectionGuard C++][election-guard-cpp-source-code]
+- [ElectionGuard Python API][election-guard-web-api-source]
+- [ElectionGuard React UI][election-guard-ui-source]
+
+This chart shows the relationship of the specification to the reference implementations by repository. This paints the picture of how each repository is related to each other.  
```

Comment:
"how each repository is related to each other" <- "how the repositories are related to each other".